apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1797,11 +1802,13 @@ tx_defaults_(channel_settle_tx = Tx, Opts, D) ->
 tx_defaults_(channel_close_mutual_tx, _Opts, _D) ->
     #{fee => default_fee(channel_close_mutual_tx)}.
 
-default_nonce(Opts, #data{opts = DOpts}) ->
-    try maps:get(nonce, Opts, maps:get(nonce, DOpts))
-    ?CATCH_LOG(_E)
-        Pubkey = maps:get(acct, Opts),
-        get_nonce(Pubkey)
+default_nonce(Opts) ->
+    case maps:find(nonce, Opts) of
```

Comment:
Although `nonce` had not been exposed to WebSocket API, fallbacking to whatever was in the open arguments is a bit dangerous and could bite us further down the road when we expose the `nonce`.

Not really an issue, just an improvement.